bnd-workspace/de.fraunhofer.abm.app/static/de.fraunhofer.abm/index.html

```diff
@@ -46,7 +46,7 @@
 				<li ng-class="{active: page() == '/login'}" ng-show="user === undefined"><a href="#/login">Login</a></li>
 				<li ng-hide="user === undefined"><a href="" ng-click="logout()">Logout</a></li>
 			</ul>
-		<h3 class="text-muted custom-right">Automatic Benchmark Management</h3>
+		<h3 class="text-muted custom-right"><br class="rwd-break">Automatic Benchmark Management</h3>
```

Comment:
The commit that replaced "Automatic" by "Automated" was made long ago. Did you update your branch before committing?